spring-integration-amqp/src/test/java/org/springframework/integration/amqp/rule/BrokerRunning.java

```diff
@@ -132,12 +132,15 @@ public Statement apply(Statement base, Description description) {
 		return super.apply(base, description);
 	}
 
-	public void removeTestQueues() {
+	public void removeTestQueues(String... additionalQueues) {
```

Comment:
How about to merge https://github.com/spring-projects/spring-integration/pull/2064 first?
We won't need this change because there won't be this local `BrokerRunning` any more.
The `spring-rabbit-junit` `BrokerRunning` has this method already.

Feel free to apply those `AmqpTests` changes during merge that PR